src/pages/documentation/adobe-media-analytics/index.md

```diff
@@ -2,6 +2,11 @@ import Tabs from './tabs/index.md'
 
 # Adobe Analytics - Media Analytics for Audio and Video
 
+<InlineAlert variant="success" slots="header, text"/>
+Media for Edge Network extension is now available! 
```

Comment:
Use the full name here, "Adobe Streaming Media for Edge Network extension is now available!"